Build/Chakra.Build.props

```diff
@@ -56,24 +49,24 @@
         %(PreprocessorDefinitions);
         CHAKRACORE_LITE
       </PreprocessorDefinitions>
-      <PreprocessorDefinitions Condition="'$(UseICU)'=='true'">
+      <PreprocessorDefinitions Condition="'$(ChakraICU)'!='false'">
         %(PreprocessorDefinitions);
         HAS_ICU;
         U_DISABLE_RENAMING=1; <!-- Disable renaming to maintain compatibility with Windows Kit ICU's icuuc/icuin.lib -->
       </PreprocessorDefinitions>
-      <PreprocessorDefinitions Condition="'$(EnableIntl)'=='true' AND '$(UseICU)'=='true'">
+      <PreprocessorDefinitions Condition="'$(EnableIntl)'=='true' AND '$(ChakraICU)'!='false'">
```

Comment:
Just so I understand- so if EnableIntl is true and ChakraICU is falsy, we use WinGlob?

Reply:
Correct

Reply:
!='false' => "truthy"?